File: SV.Web/src/app/shared/types/api/card-response.ts
import { CardType } from '../customs/card-type.enum';
import { Craft } from '../customs/craft.enum';
import { Rarity } from '../customs/rarity.enum';

export interface CardResponse {
  artLocation: string;
  craft: Craft;
  id: string;
  name: string;
  rarity: Rarity;
  type: CardType;
}

File: SV.Edge/src/SV.Edge/Controllers/Models/CardResponse.cs
using SV.Edge.Services.Constants;

namespace SV.Edge.Controllers.Models
{
    public class CardResponse
    {
        public string ArtLocation { get; init; }
        public CraftType Craft { get; init; }
        public string Id { get; init; }
        public string Name { get; init; }
        public RarityType Rarity  { get; init; }
        public CardType Type { get; init; }
    }
}